test/src/util/formatting/date-format.js

```diff
@@ -1,15 +1,15 @@
 import { expect } from 'chai';
-import { DateFormat } from 'util/formatting/date-format';
+import { DateFormat } from 'comp/util/date-format';
 
 describe('DateFormat', () => {
-    const dt = new Date(2020, 0, 2, 3, 4, 5, 6);
+    const dt = new Date.UTC(2020, 0, 2, 3, 4, 5, 6);
```

Comment:
Two things here:
1. Does this work? Date.UTC returns a number, it shouldn't be possible to use it as a constructor.
2. Printing dates in UTC is not correct, we want to display dates in the local time, not in UTC. This test must print a different value in this case. But better not to change the test.

Reply:
1)Yes, it works. I try to remember running tests before committing. I did run `grunt test`.
![image](https://user-images.githubusercontent.com/20780192/100390449-b1885f00-3030-11eb-8fd5-e70c56a7b9f1.png)
It stills work because there is a conditional if which transform all the numbers in dates.
![image](https://user-images.githubusercontent.com/20780192/100390500-e0063a00-3030-11eb-875d-6fc1123927b4.png)
2) If i do not put date into UTC it will fail due to timezone because the function uses specifically the `activeLocale`.
If we want to make it stateful, otherwise i need to change the function signature adding the requested locale. Anyway, in these tests we are trying to see if date is printed as expected, in that case we are taking `en` as Locale (the default one) and checking if it returns as should be. Isn't it?

Reply:
Ok, thanks! I'll take a look before merging, this is very strange, but I'll figure it out then.

Reply:
I see, tests are broken after webpack upgrade in `develop`, only top-level tests are run and it doesn't go inside. Ok, I'll take it from here.

Reply:
Uhmm....strange... it looks like this test is running.

Reply:
It's running, but it fails exactly where I pointed out: 
```
Uncaught TypeError: Date.UTC is not a constructor
    at Suite.<anonymous> (date-format.js:5)
```
anyway, I'll fix it and look into your test expectation, this is a framework issue, tests must not fail because of one test that throws.